README.md

```diff
@@ -9,7 +9,7 @@
 ![](https://github.com/Pyroh/Fluor/blob/master/ressources/banner.png?raw=true)
 
 # What is Fluor ?
-**Fluor** is a tool that allows you to automatically change the behaviour of the keyboard's fn keys depending on the active application. That's this simple.
+**Fluor** is a tool that allows you to automatically change the behaviour of the keyboard's fn keys depending on the active application. !t's that simple.
```

Comment:
You mean *It* right ?

Reply:
Yes. Propably should have checked one last time. Will fix it.